lib/CL/clUnloadPlatformCompiler.c

```diff
@@ -0,0 +1,32 @@
+/* OpenCL runtime library: clUnloadPlatformCompiler()
+ *
+ *    Copyright (c) 2016 Tom Gall Tampere Univ. of Tech.
```

Comment:
Perhaps the correct credits here would be "Tom Gall for Tampere Univ. of Tech." ?